ha_addon/run.sh

```
@@ -111,6 +111,7 @@ export MQTT_RESPONSE_TIMEOUT=$(bashio::config 'responseTimeout' "30")
 export POLL_CELL_DATA=$(bashio::config 'enableCellData' "false")
 export POLL_CALIBRATION_DATA=$(bashio::config 'enableCalibrationData' "false")
 export POLL_EXTRA_BATTERY_DATA=$(bashio::config 'enableExtraBatteryData' "false")
+export CELL_BALANCING_DIAGNOSTICS=$(bashio::config 'enableCellBalancingDiagnostics' "false")
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)